pgcli/main.py

```diff
@@ -79,9 +78,28 @@ def __init__(self, force_passwd_prompt=False, never_passwd_prompt=False,
         # Initialize completer
         smart_completion = c['main'].as_bool('smart_completion')
         completer = PGCompleter(smart_completion)
-        completer.extend_special_commands(CASE_SENSITIVE_COMMANDS.keys())
-        completer.extend_special_commands(NON_CASE_SENSITIVE_COMMANDS.keys())
+        completer.extend_special_commands(COMMANDS.keys())
         self.completer = completer
+        self.register_special_commands()
+
+    def register_special_commands(self):
```

Comment:
Can this be replaced with a decorator on `change_db` now? One way to avoid repetition might be to add an argument for hidden aliases in special_command directly, like
`@special_command('\c, '\c[onnect] database_name', 'Change to a new database', hidden_aliases=['use', '\connect'])